lib/reply.js

```diff
@@ -163,7 +167,14 @@ Reply.prototype.send = function (payload) {
   const hasContentType = contentType !== undefined
 
   if (payload !== null) {
-    if (typeof payload.pipe === 'function') {
+    if (
+      // node:stream
+      typeof payload.pipe === 'function' ||
+      // node:stream/web
+      typeof payload.getReader === 'function' ||
+      // Response
+      toString.call(payload) === '[object Response]'
```

Comment:
Can we not create a symbol kResponse and set it as an attribute of the payload once, so we dont need to call toString multiple times?

Reply:
I don't think we can skip it in `.send` but we can reduce one time on `onSendEnd`.